README.md

```diff
@@ -90,15 +97,20 @@ To start reading from the Safety laser scanner and publishing complete scans exe
 If you wish to set parameters from the command line, add them to the end of the command as follows: `parameter:=value`, separated by spaces.
 
 ```bash
-roslaunch psen_scan psen_scan.launch host_ip:=192.168.0.20 host_udp_port:=3050
+roslaunch psen_scan psen_scan.launch sensor_ip:=192.168.0.10 host_ip:=192.168.0.20 host_udp_port:=3050
 ```
-This example configures the Safety laser scanner to send it´s frames to 192.168.0.20:3050.
+This example configures the Safety laser scanner at 192.168.0.10 to send it´s frames to 192.168.0.20:3050.
+
+### Adjust expert parameters
+If you've created an application package with your own launch file as described in the
+[tutorials](http://wiki.ros.org/psen_scan/Tutorials/),
+you can easily adjust many other configuration parameters.
```

Comment:
many other: which ones exactly? This probably refers to the list of parameters in the above paragraph.
```suggestion
you can easily adjust the aforementioned configuration parameters.
```